modules/ROOT/pages/_partials/include-enforcement.adoc

```diff
@@ -0,0 +1,9 @@
+Based on your organization’s entitlements, Anypoint Platform enforces limits for vCore consumption, network connections (VPNs and transit gateway attachments), and private spaces.
+
+CloudHub 1.0 and CloudHub 2.0 share entitlement quotas. If your organization uses both CloudHub 1.0 and CloudHub 2.0, you must budget your consumption with both in mind. For example, if your organization is entitled to four network connections, it can support one transit gateway connection in CloudHub 1.0, one transit gateway connection in CloudHub 2.0, and two VPNs (or some combination therein). In other words, if the network connection entitlement is four, you do not get four network connections for each version of CloudHub. To ensure your organization has sufficient entitlements to meet your needs, you must understand how resources map to entitlements:
+
+* Private cloud quotas include both VPCs (virtual private clouds) in CloudHub 1.0 and private spaces in CloudHub 2.0.
+* Network connection quotas include VPNs and transit gateway attachments in both CloudHub 1.0 and CloudHub 2.0.
+* vCore usage quotas apply to apps deployed in both CloudHub 1.0 and CloudHub 2.0.
+
+Anypoint Platform prevents you from consuming resources that exceed your organization’s limits. Anypoint Platform also prevents you from modifying existing resources if you are exceeding that resource’s limit. Note that failed application deployments and unsuccessful private space creation in CloudHub 2.0 still count toward limits.
```

Comment:
`Note that failed application deployments and unsuccessful private space creation in CloudHub 2.0 still count toward limits.`

@hegderohit89 @MichelleZMY is it correct?

Reply:
that's my understanding based on ump framework. until user specifically delete the app/ps deployment they'd count toward usage.

Reply:
+1, the model we have currently keeps the failed deployments/spaces as the way for users to learn about the failures.
Its then expected that user will take some action on that to either fix the issue/delete the deployments.
Until then from UMP's perspective the quota is utilized.
Tagging @tomasmussi-mulesoft  to confirm this ^^

Reply:
Yes, this is correct. Failed deployments still count quota